tests/PHPUnit/System/ArchiveInvalidationTest.php

```diff
@@ -201,7 +203,35 @@ public function testDisablePluginArchiveBySiteId()
         $this->assertFalse(Rules::isSegmentPluginArchivingDisabled('testPlugin', 1));
     }
 
+    /**
+     * Check that both date range types are correctly parsed without any exceptions
+     */
+    public function testDateRangesCorrectlyParsed()
+    {
+        $testRanges = [
+            '2020-01-01,2020-03-31' => '2020-01-01,2020-03-31',
```

Comment:
It seems we are currently not testing those methods well. Might be good to also add tests for other possible cases other than the new range, like providing a single day or month or a list of days

Reply:
Makes sense :+1: I've adjusted the tests to cover specific date ranges, lastN ranges, a single specific day, a single month, a list of days and a list of today/yesterday days.